app/src/main/java/xterminators/spellingbee/model/HighScores.java

```diff
@@ -0,0 +1,148 @@
+package xterminators.spellingbee.model;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+
+import java.util.Scanner;
+import java.util.TreeMap;
+import java.util.Map;
+import java.util.Collections;
+
+import java.lang.reflect.Type;
+
+import com.google.gson.Gson;
+import com.google.gson.reflect.TypeToken;
+
+public class HighScores {
+
+    private static final int MAX_STORED_SCORES = 10;
+
+    // Use TreeMap so elements are ordered
+    private static TreeMap<String, Integer> scores;
```

Comment:
I'd use Map as the type here, so that we have data structure flexibility. Also I'd use a HashMap as we don't care about order (on keys).